src/vs/workbench/contrib/chat/browser/chatListRenderer.ts

```diff
@@ -596,6 +597,12 @@ class CodeBlockPart extends Disposable implements IChatResultCodeBlockPart {
 			}
 		}));
 
+		if (this._accessibilityService.isScreenReaderOptimized()) {
+			const toolbarElt = this.toolbar.getElement();
+			toolbarElt.style.display = 'block';
```

Comment:
This would require you to reload the window for this to actually take effect on a rendered codeblock, is that ok?

Reply:
In OSS, that didn't appear to be the case 🤔

Reply:
Oh yeah but usually a window will start w accessibility mode enabled

Reply:
Because of the way we detect that. But sure, we can listen for config changes too. I wasn't sure if that was overkill.

Reply:
Up to you, I was just checking 🤷

Reply:
fixed 👍🏼